### PR TITLE
DC 35 i18n setup 

### PR DESCRIPTION
**Pull request by MarcoGoC**
### Description
    Setting up i18n added DC-34, Ben's Layout and Footer code, as well as some fixes for the splash page
    there still some issues on the splash page to be solved  

List of proposed changes:
- Setting up i18n on the next.config
- Added language translation on the Layout component
- Added language translation files (en and fr) under the locales folder
- re-added AEM code on the home page 
- fixed the splash background image on both index.js and the tailwind.config

### Addtional Notes
- TODO Text props for Modified Date needs to be passed from the footer

### CODE REVIEW

File: locales/fr.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  reportAProblemNoPersonalInfo:
+    "Ne pas inclure de renseignements personnels, médicaux ou financiers, comme par exemple, un numéro d'assurance sociale (NAS), une adresse de domicile ou de lieu de travail, un numéro de téléphone ou numéro de dossier.",
+  reportAProblemThankYouForYourHelp: 'Merci pour votre aide!',
+  reportAProblemYouWillNotBeContacted:
+    'Vous ne recevrez pas de réponse. Pour toute demande de renseignements, veuillez nous contacter à',
+  reportAProblemPrivacyStatement: 'Déclaration de confidentialité',
+  reportAProblemPrivacyStatementLink:
+    'https://www.canada.ca/fr/transparence/confidentialite.html',
+  reportAProblemSubmit: 'Soumettre',
+}
```
Comment (DaceyTom2) [inline locales/fr.js:124]: If these aren't used can they be removed?

File: next.config.js [hunk 2]
```diff
@@ -18,4 +18,27 @@ module.exports = {
...
+    defaultLocale: 'en',
+    localDetection: true,
+  },
+  //
+  // rewrites setup
+  //
+  async rewrites() {
+    return [
+      {
+        source: '/accueil',
```
Comment (DaceyTom2) [inline next.config.js:35]: This looks to work well. Seems like /fr/home still works. Any chance we could do a demo?

File: locales/en.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  reportAProblemNoPersonalInfo:
+    'Do not include any personal, medical, or financial details, such as your name, social insurance number (SIN), home or business address, phone number or any case or file numbers.',
+  reportAProblemThankYouForYourHelp: 'Thank you for your help!',
+  reportAProblemYouWillNotBeContacted:
+    'You will not receive a reply. For enquiries, please contact us at ',
+  reportAProblemPrivacyStatement: 'Privacy statement',
+  reportAProblemPrivacyStatementLink:
+    'https://www.canada.ca/en/transparency/privacy.html',
+  reportAProblemSubmit: 'Submit',
+}
```
Comment (DaceyTom2) [inline locales/en.js:122]: If these aren't used can they be removed?

File: components/atoms/ActionButton.js [hunk 4]
```diff
@@ -0,0 +1,192 @@
+import PropTypes from 'prop-types'
```
Comment (DaceyTom2) [inline components/atoms/ActionButton.js:1]: This button isn't being used in this PR

**Review by DaceyTom2 — DISMISSED** [review]

File: next.config.js (hunk 2 above)
Comment (MarcoGoC) [reply next.config.js:35]: yes, this is part of DC-34

File: locales/en.js (hunk 3 above)
Comment (MarcoGoC) [reply locales/en.js:122]: yes

File: components/atoms/ActionButton.js (hunk 4 above)
Comment (MarcoGoC) [reply components/atoms/ActionButton.js:1]: it was on dc-34 but not needed with latest changes for the splash page

Comment (MarcoGoC) [reply components/atoms/ActionButton.js:1]: removed

**Review by nibivi77 — APPROVED** [review]
This is great! The only thing I think we are all concerned about is the Action Button being really big and unused for the most part. Once we need it, we should refactor it. For now, this is good.

**Pull request merged**